packages/cosmic-swingset/README.md

```diff
@@ -277,10 +171,6 @@ home.uploads~.get('encouragementBot')~.spawn()~.encourageMe('Person')
 ### Initial Endowments
 
 When a client is started up, it has a few items in a record named home.
-
-* gallery: the Pixel Gallery, described above
-* purse: a purse that can hold pixel Dust
-* moolah: a purse that starts out with 1000 `moolah`
```

Comment:
We will still make a moola[h] purse, I think. Does it not end up in `home`?

Reply:
The wallet has all the purses now!